torch/csrc/distributed/rpc/request_callback_no_python.cpp

```diff
@@ -147,22 +147,19 @@ void RequestCallbackNoPython::processScriptCall(
   processScriptCallOp(scriptCall, markComplete, stack);
 }
 
-bool RequestCallbackNoPython::processScriptCallOp(
+void RequestCallbackNoPython::processScriptCallOp(
     ScriptCall& scriptCall,
     const std::function<void(Message)>& markComplete,
     std::vector<at::IValue>& stack) const {
-  if (scriptCall.hasOp()) {
-    scriptCall.op()->getOperation()(&stack);
-    TORCH_INTERNAL_ASSERT(
-        stack.size() == 1,
-        "Return value of a builtin operator or a "
-        "TorchScript function should be a single IValue, got a vector of "
-        "size ",
-        stack.size());
-    markComplete(std::move(ScriptResp(std::move(stack.front()))).toMessage());
-    return true;
-  }
-  return false;
+  TORCH_INTERNAL_ASSERT(scriptCall.hasOp());
+  scriptCall.op()->getOperation()(&stack);
+  TORCH_INTERNAL_ASSERT(
+      stack.size() == 1,
+      "Return value of a builtin operator or a "
+      "TorchScript function should be a single IValue, got a vector of "
+      "size ",
```

Comment:
nit: these can fit in two lines?

Reply:
Please feel free to update this in followup PRs. I don't want to make you rebase and wait for another 6H, just for these little changes.